packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -177,41 +177,50 @@ class RequestableSizeWidget extends StatelessWidget {
 /// Wrapper to get the name of an ObjRef depending on its type.
 String? _objectName(ObjRef? objectRef) {
   String? objectRefName;
-  if (objectRef == null) return null;
 
-  if (objectRef is ClassRef ||
-      objectRef is FuncRef ||
-      objectRef is FieldRef ||
-      objectRef is LibraryRef) {
+  if (objectRef is ClassRef || objectRef is FuncRef || objectRef is FieldRef) {
     objectRefName = (objectRef as dynamic).name;
+  } else if (objectRef is LibraryRef) {
+    objectRefName =
+        objectRef.name?.isEmpty ?? false ? objectRef.uri : objectRef.name;
+  } else if (objectRef is ScriptRef) {
+    objectRefName = fileNameFromUri(objectRef.uri);
   } else if (objectRef is InstanceRef) {
     objectRefName = objectRef.name ??
         'Instance of ${objectRef.classRef?.name ?? '<Class>'}';
+  } else {
+    objectRefName = objectRef?.vmType;
   }
 
   return objectRefName;
 }
 
-/// Recursively gets the full owner name of a field or function object.
-String? _ownerName(ObjRef? ref) {
+/// Recursively gets the name of a function, qualified with its full owner name,
```

Comment:
what does 'qualified' mean in this context? can you elaborate?

Reply:
For example: for the function build with owner GalleryApp, the qualified name would be GalleryApp.build. 
These are some other examples:
<img width="1087" alt="image" src="https://user-images.githubusercontent.com/105914998/181647137-5f507f7a-d610-429f-9e8e-e74277f1ae54.png">

Reply:
can you add this context to the dart doc? thanks

Reply:
Will do